.circleci/config.yml

```diff
@@ -60,7 +60,7 @@ jobs:
             0.0.0.0       rabbitmq.local
             ' | tee -a /etc/hosts
 
-      - run: gem install bundler
+      - run: gem install bundler -v '~> 1.17.0'
```

Comment:
does this also update rubyGems to >= 3.0.0?

Reply:
No. It just specifies the version of `bundler` to install, which is the same version that we've been using. If we update RubyGems, then we can go back to not specifying the `bundler` version. I decided to go this route because the version of RubyGems in the `ruby:2.3.6` docker image is `2.7.6` and updating that would probably mean updating ruby, which is much more involved than setting the `bundler` version.

Reply:
ok.